infrastructures/kubernetes/src/main/java/org/eclipse/che/workspace/infrastructure/kubernetes/namespace/KubernetesIngresses.java

```diff
@@ -139,6 +142,18 @@ public void onClose(WatcherException cause) {
 
   public void delete() throws InfrastructureException {
     try {
+
+      clientFactory
+          .create(workspaceId)
+          .network()
+          .v1()
+          .ingresses()
+          .inNamespace(namespace)
+          .withLabel(CHE_WORKSPACE_ID_LABEL, workspaceId)
+          .withPropagationPolicy(BACKGROUND)
+          .delete();
+
+      // compatibility
```

Comment:
so old ingresses are not deleted if you remove this code?

Reply:
If yes, can you please write in the comment what's the background story of why we need to keep it and when it will be possible to remove?

Reply:
looks like we can omit that.. at least on my latest minikube it works. lets try with that.